website/docs/limitations.md

```diff
@@ -0,0 +1,28 @@
+---
+id: limitations
+title: Limitations
+description: A limitations table where every technical and business limitation is collected for each subscription plan.
+---
+
+## A limitations table where every technical and business limitation is collected for each subscription plan.
+
+:::info Every bolded value in the table marks a technical limitation. Meaning that the value can be increased by [contacting us](https://configcat.com/support/).
+:::
+
+|                                                          | Free  | Pro   | Smart    | Enterprise | Dedicated | Student | Teacher |
+| -------------------------------------------------------- | ----- | ----- | -------- | ---------- | --------- | ------- | ------- |
+| Config JSON downloads  (million)                         | 5     | 25    | 250      | 1000       | 5000      | 25      | 5       |
+| Network traffic (GB/month)                               | 20    | 100   | 1024     | 4096       | 24576     | 100     | 100     |
```

Comment:
I'd go with the way we show this on the pricing table - not converting everything to GB but 
e.g. 20GB | 100 GB | 1 TB | 4 TB | 24 TB | 100GB | 100GB.
24576GB seems a bit overwhelming for me.